protect/Dockerfile.template

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:14.04
+
+# File Author / Maintainer
+MAINTAINER John Vivian <jtvivian@gmail.com>
+
+RUN apt-get update && apt-get install -y \
+    git \
+    python-dev \
+    python-pip \
+    wget \
+    curl \
+    apt-transport-https \
+    ca-certificates
+
+# Get the Docker binary
+RUN wget https://get.docker.com/builds/Linux/x86_64/docker-DOCKERVER -O /usr/local/bin/docker
```

Comment:
Can you use the Docker download command from #214 and use it here? The newer version of docker changed the way they distribute binaries & this wont work for those new versions